src/devrules/cli_commands/config_edit.py

```
@@ -0,0 +1,383 @@
+
+        prompter.info(f"Path: {key_path}")
+        prompter.info(f"Old Value: {old_value}")
+        prompter.info(f"New Value: {typed_value} ({type(typed_value).__name__})")
+
+        if prompter.confirm("Save changes?"):
+            try:
+                _set_value(config, key_path, typed_value)
+                _save_config(path, config)
+            except ValueError as e:
+                prompter.error(str(e))
+                raise prompter.exit(code=1)
```

Ruff (0.14.13)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)